deploy_js_lambda.sh

```diff
@@ -0,0 +1,32 @@
+#/bin/bash
+
+STAGE=$1
+BUCKET=$2
```

Comment:
Do we need this parameter? The bucket for the lambda is currently [hard coded](https://github.com/guardian/grid-infra/blob/master/platform/cloud-formation/media-service.json#L3684) in the template.